docs/source/install/upgrades.rst

```diff
@@ -198,6 +198,24 @@ v3.4
 
     sudo yum install python3-devel --enablerepo <optional-server-rpm repo>
 
+* *Ubuntu 16.04 Xenial only.* Python 3.6 is not available in the base Ubuntu Xenial distribution and you can add unofficial 3rd party `Python PPA repository <https://launchpad.net/~deadsnakes/+archive/ubuntu/ppa>`_:
+
+  .. warning::
+
+     Please be aware of the support and security risks associated with using unofficial 3rd party PPA repository.
+     StackStorm does NOT provide ANY support or security update for python3.6 packages on Ubuntu 16.04.
+     If security is a priority for you, we recommend starting migrating to Ubuntu 18.04 LTS (Bionic) as a base OS which has official python 3.6 packages.
+     This is a workaround to support Ubuntu Xenial with python 3 until we deprecate it in the future versions.
```

Comment:
A question really / but I wondered why the apt-get show and the extra comment about this being workaround before xenial is dropped were only on the upgrade and not install page.
Just checking this was on purpose, rather than an oversight.

Reply:
Right and good catch!

I thought that upgrade instructions should include a bit more info for the users to verify if the migration went well, while for raw installation instructions we just care about the end result without any debugging or transition steps or info.
That was an idea behind it.

Reply:
That's good - makes sense, just wanted to double check